lib/blockscore/base.rb

```diff
@@ -33,12 +33,12 @@ def inspect
 
     def save
       save!
-    rescue BlockScore::Error
+    rescue Error
       false
     end
 
     def save!
-      fail BlockScore::Error, "#{resource} is already deleted" if deleted?
+      assert_not_deleted
       capture_attributes(post(endpoint, attributes))
```

Comment:
> I am not sure what you mean by this - could you explain further?

It's demonstrated by the failures mutant is pointing out. Right now mutant can change:

```ruby
capture_attributes(post(endpoint, attributes))
```

To:

```ruby
capture_attributes(post(endpoint, nil))
```

.. without causing a spec failure.

This is *not* a flaw in mutant. Mutant is demonstrating a flaw in the specs. The specs depend on external state, which by definition makes them not unit tests but rather integration tests. I would be more likely to create a `specs/integration` folder and move any vcr (or external service) dependent specs into them and then create a mock connection object for true unit tests.